app/templates/events/view.hbs

```diff
@@ -20,14 +20,16 @@
       <div class="twelve wide column {{unless device.isMobile 'right aligned'}}">
         {{#if device.isMobile}}
           <div class="ui icon fluid buttons">
-            <a href="{{href-to 'public' model.id}}" class="ui button">
+            <a href="{{href-to 'public' model.id}}" class="ui button" data-tooltip="Preview" data-position="top left">
```

Comment:
use translation helpers for all the tool-tips.

Reply:
@CosmicCoder96 Done the changes , Review now